CalibTracker/SiStripAPVAnalysis/src/ApvAnalysisFactory.cc

```diff
@@ -118,7 +118,10 @@ void ApvAnalysisFactory::constructAuxiliaryApvClasses(ApvAnalysis* theAPV, uint3
       theCM = new MedianCommonModeCalculator();
     } else {
       cout << "Sorry Only Median is available for now, Mean and FastLinear are coming soon" << endl;
+      return;
     }
+  } else {
+    return;
```

Comment:
If one returns here, probably better not doing it silently. A `cout`, as in previous [L120](https://github.com/cms-sw/cmssw/pull/46074/files#diff-c191c0b9b0798dc37092e887ee8affa0ee4a0f774d4a96e1d6925caaa9e03fcdR120), could work:
```suggestion
  } else {
    cout << "ApvAnalysisFactory: algorithm " << theAlgorithmType_ << " not supported" << endl:
    return;
```